test/compile.jl

```diff
@@ -32,8 +39,12 @@ try
 
     # use _require_from_serialized to ensure that the test fails if
     # the module doesn't load from the image:
-    println(STDERR, "\nNOTE: The following 'replacing module' warning indicates normal operation:")
```

Comment:
this warning is probably worth keeping? or the error redirection isn't complete...